tests/wpt/metadata/html/semantics/forms/textfieldselection/select-event.html.ini

```diff
@@ -1,23 +1,146 @@
+    expected: FAIL
+
+  [input type text: setSelectionRange() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type text: setRangeText()]
+    expected: FAIL
+
+  [input type text: setRangeText() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: select()]
+    expected: FAIL
+
+  [input type search: select() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: selectionStart a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: selectionEnd a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: selectionDirection a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: setSelectionRange() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type search: setRangeText()]
+    expected: FAIL
+
+  [input type search: setRangeText() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: select()]
+    expected: FAIL
+
+  [input type tel: select() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: selectionStart a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: selectionEnd a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: selectionDirection a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: setSelectionRange() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type tel: setRangeText()]
+    expected: FAIL
+
+  [input type tel: setRangeText() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: select()]
+    expected: FAIL
+
+  [input type url: select() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: selectionStart a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: selectionEnd a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: selectionDirection a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: setSelectionRange() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type url: setRangeText()]
+    expected: FAIL
+
+  [input type url: setRangeText() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: select()]
+    expected: FAIL
+
+  [input type password: select() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: selectionStart a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: selectionEnd a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: selectionDirection a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: setSelectionRange() a second time (must not fire select)]
+    expected: FAIL
+
+  [input type password: setRangeText()]
+    expected: FAIL
+
```

Comment:
Why are there so many expected test failures?

Reply:
These are just things that haven't yet been implemented:

* `select()` method
* `setRangeText()` method
* Code to avoid firing the `select` event when the selection hasn't actually moved